docs/guide/slots.md

```diff
@@ -6,28 +6,52 @@ parent: Building ViewComponents
 
 # Slots
 
-In addition to the `content` accessor, ViewComponents accept content through Slots, enabling multiple blocks of content to be passed to a single ViewComponent.
+In addition to the `content` accessor, ViewComponents accept content through slots, enabling multiple blocks of content to be passed to a single ViewComponent.
 
 Slots are defined with `renders_one` and `renders_many`:
 
 - `renders_one` defines a slot that will be rendered at most once per component: `renders_one :header`
-- `renders_many` defines a slot that can be rendered multiple times per-component: `renders_many :blog_posts`
+- `renders_many` defines a slot that can be rendered multiple times per-component: `renders_many :posts`
 
-_To view documentation for content_areas (deprecated) and the original implementation of Slots (deprecated), see [/content_areas](/content_areas) and [/slots_v1](/slots_v1)._
+For example:
 
-## Defining slots
+```ruby
+# blog_component.rb
+class BlogComponent < ViewComponent::Base
+  renders_one :header
+  renders_many :posts
+end
+```
 
-_In versions `< 2.28.0`, `include ViewComponent::SlotableV2` to use slots._
+To render a `renders_one` slot, call the name of the slot.
 
-Slots come in three forms:
+To render a `renders_many` slot, iterate over the name of the slot:
 
-- [Delegate slots](#delegate-slots) render other components.
-- [Lambda slots](#lambda-slots) render strings or initialized components.
-- [Pass through slots](#pass-through-slots) pass content directly to another component.
+```erb
+<%# blog_component.html.erb %>
+<h1><%= header %></h1>
 
-## Delegate slots
+<% posts.each do |post| %>
+  <%= post %>
+<% end %>
+```
 
-Delegate slots delegate to another component:
+```erb
+<%# index.html.erb %>
+<%= render BlogComponent.new do |c| %>
+  <% c.header(classes: '') do %>
```

Comment:
@joelhawksley a bit late since you already merged, but I find this `classes: ''` confusing...

Reply:
@Spone hmm. What might be a better name?